### PR TITLE
[BUG] Stock Refund: Correção quando se tenta retornar alterando as quantidades ou retirando um item do popup.

### PR DESCRIPTION
**Pull request by mileo**

### CODE REVIEW

File: l10n_br_stock/wizard/stock_return_picking.py [hunk 1]
```diff
@@ -39,68 +39,69 @@ def create_returns(self, cr, uid, ids, context=None):
          @param context: A standard dictionary
          @return: A dictionary which of fields with values.
         """
+        result = super(stock_return_picking, self).create_returns(
+            cr, uid, ids, context)
+
+        data = self.read(cr, uid, ids[0])
```
Comment (bwrsandman) [inline l10n_br_stock/wizard/stock_return_picking.py]: Wouldn't this be more efficient?

``` python
self.read(cr, uid, ids[0], ['invoice_state'], context=context)
```

File: l10n_br_stock/wizard/stock_return_picking.py [hunk 2]
```diff
@@ -39,68 +39,69 @@ def create_returns(self, cr, uid, ids, context=None):
...
+                                'picking_type' : picking_type,                                    
+                                })
+                fiscal_position = move_obj.onchange_product_id(cr, uid, ids, move.product_id.id, move.location_id.id,
+                         move.location_dest_id.id, picking.partner_id.id, context)['value'].get('fiscal_position') or False
+                fiscal_category_id = move_fc_return_id or False
+                  
+                move_obj.write(cr, uid, move.id,{'fiscal_position': fiscal_position,
+                                                        'fiscal_category_id': fiscal_category_id,
+                                                        },context)
+        return result
```
Comment (bwrsandman) [inline l10n_br_stock/wizard/stock_return_picking.py]: No newline at the end of the file means pep8 error

File: l10n_br_stock/wizard/stock_return_picking.py (hunk 1 above)
Comment (mileo) [inline l10n_br_stock/wizard/stock_return_picking.py]: :+1:

**Comment by bwrsandman** [comment]
Looks good to me! :+1:

**Comment by mileo** [comment]
:100: UP!!! jul 10

**Comment by legalsylvain** [comment]
@rvalyi could you take a look on that old PR ?

**Comment by rvalyi** [comment]
@mileo @danimaribeiro 
sorry that was during the bad months here. You still think this fix should be applied?
cc @renatonlima

**Pull request closed without merging**